docs/antora-ccutil/antora-playbook-ccutil.yml

```diff
@@ -0,0 +1,57 @@
+site:
+  title: Keycloak Docs
+  # the 404, site-map, and canonical urls are only generated when an url is defined
+  url: http://localhost:5000/keycloak-antora/
+  start_page: guides-server::index.adoc
+content:
+  sources:
+    - url: ../..
+      start_paths:
+        - docs/antora-ccutil/components/guides-server
+        - docs/antora-ccutil/components/guides-operator
+      branches:
+        - 'HEAD'
+
+urls:
+  latest_version_segment: latest
+  html_extension_style: drop
+
+antora:
+  extensions:
+    - '@antora/collector-extension'
+    - './lib/index.js'
+
+ui:
+  bundle:
+    url: https://gitlab.com/antora/antora-ui-default/-/jobs/artifacts/HEAD/raw/build/ui-bundle.zip?job=bundle-stable
+    # snapshot: true
+  supplemental_files: supplemental_ui
+  output_dir: antora
+
+runtime:
+  fetch: true
+  cache_dir: target/antora/cache
+  log:
+    failure_level: error
+    format: pretty
+
+output:
+  clean: true
+  dir: target/site/keycloak-antora
+
+# Global asciidoc attributes here, used across all versions of documentation
+asciidoc:
+  attributes:
+    idprefix: ''
+    idseparator: '-'
+    experimental: ''
+    page-pagination: ''
+    project_name: Keycloak
+    project_name_full: Keycloak
+    project_version: 20.0.0
+    project_versionMvn: 20.0.0
+    project_versionNpm: 20.0.0
+    project_versionDoc: 20.0
+    adminguide_name: Server Administration Guide
+    adminguide_name_short: Server Administration
```

Comment:
Are these needed? Ideally I don't want to have the version in many files as it makes it hard to maintain.

Reply:
Will later be sourced from a `versions.adoc` file from the Freemarker output.